server/src/Administration/Infrastructure/Finders/Doctrine/DoctrineCompanyFinder.php

```diff
@@ -30,85 +31,62 @@ public function __construct(Connection $connection)
 
     /**
      * @throws Exception
-     * @throws \Doctrine\DBAL\Exception
      */
-    public function findByUuid(string $uuid): CompanyModel
+    public function findByUuid(string $uuid): CompanyReadModel
     {
-        $query = <<<'SQL'
-SELECT
-    company.uuid as uuid,
-    company.name as name,
-    company.address as address,
-    company.zip_code as zipCode,
-    company.town as town,
-    company.country as country,
-    company.phone as phone,
-    company.facsimile as facsimile,
-    company.email as email,
-    company.contact_name as contactName,
-    company.cellphone as cellphone,
-    company.slug as slug
-FROM company
-WHERE uuid = :uuid
-SQL;
-        $result = $this->connection->executeQuery($query, ['uuid' => $uuid])->fetchOne();
+        $query = $this->connection->createQueryBuilder()
+            ->select(
+                'uuid',
+                'name',
+                'address',
+                'zip_code',
+                'town',
+                'country',
+                'phone',
+                'facsimile',
+                'email',
+                'contact_name',
+                'cellphone',
+                'slug'
+            )
+            ->from('company')
+            ->where('uuid = ?')
```

Comment:
Personal preference: named parameters instead of "?". It will improve the readability for sure